src/hotspot/share/runtime/javaThread.hpp

```diff
@@ -159,7 +160,18 @@ class JavaThread: public Thread {
   // One-element thread local free list
   JNIHandleBlock* _free_handle_block;
 
+  // ID used as owner for inflated monitors. Same as the j.l.Thread.tid of the
+  // current _vthread object, except during creation of the primordial and JNI
+  // attached thread cases where this field can have a temporal value.
```

Comment:
```suggestion
  // attached thread cases where this field can have a temporary value.
```
Presumably this is for when the attaching thread is executing the Thread constructor?

Reply:
Exactly.

Reply:
Comment adjusted.